frontend/test/metabase/scenarios/visualizations/reproductions/16170-line-mongo-replace-missing-values.cy.spec.js

```diff
@@ -34,8 +34,8 @@ describe("issue 16170", () => {
       cy.get(".dot").eq(-2).trigger("mousemove", { force: true });
 
       popover().within(() => {
-        testPairedTooltipValues("Created At", "2018");
-        testPairedTooltipValues("Count", "6,578");
+        testPairedTooltipValues("Created At", "2019");
+        testPairedTooltipValues("Count", "6,524");
```

Comment:
This is the value we get if we are in the `America/Los_Angeles` time zone.

Reply:
@losrebellos Could you please have a look at this change?

Reply:
It seems that this specific test is checking what appears in the Tooltip.
Does it look correct @nemanjaglumac?

Reply:
@nemanjaglumac Could you please have a look at this change?

Reply:
It is testing the tooltip values, yes.
I'll check why the previous values didn't work.

Btw, the timezone for Cypress is set here:
https://github.com/metabase/metabase/blob/master/frontend/test/__runner__/cypress-runner-backend.js#L29